python/ql/lib/change-notes/2025-02-11-fix-match-literal-pruning.md

```diff
@@ -0,0 +1,5 @@
+---
+category: fix
+---
+
+- `MatchLiteralPattern`s are now never pruned, as this could lead to code being wrongly identified as unreachable.
```

Comment:
Can we add an example so the reader knows what these look like in code?
```suggestion
- `MatchLiteralPattern`s such as `...` are now never pruned from the extracted source code, as this could lead to code being wrongly identified as unreachable.
```

Reply:
Done! Sorry for the delay.